### ISSUE TITLE
List of all published Galactic News

### ISSUE THREAD

**Issue by beatrizserrano** [galaxyproject/galaxy-hub#1099]
@erxleben is looking at the past Galactic News, and we can't find a list that gathers all the published ones. Does this exist?

We think that it would be useful to have something similar to this: https://galaxyproject.org/community/devroundtable/ 
I guess that a new section under Community (https://galaxyproject.org/community/newsletter/) would work. It would be just a filtered version of the News, and we could add general information, like how to subscribe, the frequency, how to contribute content (linking to the new form), etc.

**Comment by beatrizserrano** [comment]
oh I just discovered this: https://galaxyproject.org/galaxy-updates/

can be that this has been manually updated?

**Comment by NickSto** [comment]
Yeah that page isn't automated right now. I remember mentioning the idea of automating it to Dave but he didn't seem to think it was a high priority?

But it's not Dave's job anymore so if you guys would like an automated page then let's do it.

**Comment by beatrizserrano** [comment]
Not high priority I think, but something to keep in mind :)
